ui/src/org/pentaho/di/ui/core/widget/TextVar.java

```diff
@@ -153,7 +153,7 @@ public void setInsertTextInterface( InsertTextInterface insertTextInterface ) {
     this.insertTextInterface = insertTextInterface;
   }
 
-  private ModifyListener getModifyListenerTooltipText( final Text textField ) {
+  protected ModifyListener getModifyListenerTooltipText( final Text textField ) {
     return new ModifyListener() {
       public void modifyText( ModifyEvent e ) {
         if ( textField.getEchoChar() == '\0' ) { // Can't show passwords ;-)
```

Comment:
Is it possible that without the change to the echo character that it used to do, that if you hover over a password field, you'll see the password ?